setup/ubuntu14.04/ansible-vars.yaml

```diff
@@ -2,10 +2,10 @@
 server_user: iojs
 ssh_users:
   - rvagg
-  - wblankenship
-  - ryanstevens
   - mhdawson
   - joaocgreis
+  - jbergstroem
+  - orangemocha
 packages:
```

Comment:
Same as above (downloading pubkeys). We don't really use per-user keys any longer. We could always walk through all files and remove all things related to pubkeys but that means we'd have to replace it with the new system which I didn't want to spend too much time on outside of the `2.0` refactor.